test/e2e/gatekeeper_controller_test.go

```diff
@@ -557,6 +557,8 @@ var _ = Describe("Gatekeeper", func() {
 	Describe("Test in Openshift Env", Label("openshift"), Ordered, Serial, func() {
 		const openshiftRoutePath = "../resources/gatekeeper_test/openshift-route-crd.yaml"
 		const openshiftNamespace = "openshift-gatekeeper-system"
+		const openshiftSecretName = "gatekeeper-webhook-server-cert-ocp"
```

Comment:
Why not use `controller.OpenshiftSecretName` here, since it's a published variable?